bin/ch/ch.cpp

```diff
@@ -94,7 +94,12 @@ void __stdcall PrintUsage()
 
 void __stdcall PrintChVersion()
 {
-    wprintf(_u("%s version %d.%d.%d.0\n"), hostName, CHAKRA_CORE_MAJOR_VERSION, CHAKRA_CORE_MINOR_VERSION, CHAKRA_CORE_PATCH_VERSION);
+#if CHAKRA_CORE_VERSION_RELEASE
+    wprintf(_u("%s version %d.%d.%d.0\n"),
+#else
+    wprintf(_u("%s version %d.%d.%d.0-beta\n"),
```

Comment:
>-beta [](start = 37, length = 5)

Have you synced this "-beta" suffix with others (consistency with messages / terms elsewhere, if we publish prereleases?)